src/renderer/pages/project/chat.vue

```
@@ -264,6 +272,21 @@
       config,
     });
     graphai.registerCallback(streamPlugin(streamNodes));
+    // graphai.registerCallback(console.log);
+    graphai.registerCallback((data) => {
+      const { agentId, nodeId, state, result, namedInputs } = data;
+      if (nodeId === "toolCallAgent" && state === "completed") {
+        const input = data["inputsData"].find((element) => element.arguments);
```

ESLint

[error] Remove this useless assignment to variable "input".

(sonarjs/no-dead-store)